consul: Remove implicit workload identity when task has a template. (#25298)
hashicorp/nomad: website/content/docs/job-specification/identity.mdx

  readable by that user. Otherwise the file is readable by everyone but is
  protected by parent directory permissions.
- `filepath` `(string: "")` - If not empty and file is `true`, the workload

>   identity will be available at the specified location relative to the

Reviewer: ```suggestion
  identity is available at the specified location relative to the
```
